cmd/cb-event-forwarder/amqp.go

```diff
@@ -62,13 +63,14 @@ func NewConsumer(amqpURI, queueName, ctag string, bindToRawExchange bool,
 		return nil, nil, fmt.Errorf("Channel: %s", err)
 	}
 
+	durable := config.AMQPDurableQueues
```

Comment:
Three discussion points:

1. How do we name our queues so that they 'follow us'? The queues used to be named by PID, so we would create a new queue every time it re-launches - with a durable queue we need to make sure the queue name stays stable, instead
2. We need a way to 'clean up' queues we no longer need
3. We need to declare a maximum size for the queues so we don't accidentally bring down Cb Response servers with massive queue backlogs (say if we experiment with the event forwarder, then never run it again...)